src/provider/ehr_launch.jl

```diff
@@ -164,10 +164,11 @@ function provider_ehr_launch_part_two(
         "GET", authorize_uri_with_querystring_params; redirect=false
     )
     headers = Dict(authorize_response.headers)
+    redirect_uri = config.redirect_uri
     let
         error_msg = string(
             "Something went wrong when authenticating to the EHR. ",
-            "One possible explanation is that your `redirect_uri` value does not exactly ",
+            "One possible explanation is that your `redirect_uri` value (`$(redirect_uri)`) does not exactly ",
```

Comment:
String interpolation could be avoided here (along the lines of https://docs.julialang.org/en/v1/manual/performance-tips/#Avoid-string-interpolation-for-I/O):
```suggestion
            "One possible explanation is that your `redirect_uri` value (`",
            redirect_uri,
            "`) does not exactly ",
```